airbyte-integrations/connectors/source-camunda-history/source_camunda_history/spec.yaml

```
@@ -0,0 +1,40 @@
+connectionSpecification:
+  type: object
+  $schema: http://json-schema.org/draft-07/schema#
+  required:
+    - username
+    - start_date
+    - batchsize
+    - baseurl
+  properties:
+    baseurl:
+      type: string
+      order: 4
+      title: baseurl
+      description: http://localhost:8090
```

```suggestion
    baseurl:
      type: string
      order: 4
      title: Camunda History URL
      description: Camunda History API endpoint
      example: https://localhost:8090
```

Can you improve the description, title and make order organized. Thanks

---

yes, updating